inc/class-core-sitemaps-taxonomies.php

```diff
@@ -64,42 +64,52 @@ public function get_url_list( $page_num ) {
 
 		$url_list = array();
 
+		// Offset by how many terms should be included in previous pages.
+		$offset = ( $page_num - 1 ) * CORE_SITEMAPS_POSTS_PER_PAGE;
+
 		$args = array(
-			'fields'     => 'ids',
-			'taxonomy'   => $type,
-			'orderby'    => 'term_order',
-			'number'     => CORE_SITEMAPS_POSTS_PER_PAGE,
-			'paged'      => absint( $page_num ),
-			'hide_empty' => true,
+			'taxonomy'     => $type,
+			'orderby'      => 'term_order',
+			'number'       => CORE_SITEMAPS_POSTS_PER_PAGE,
+			'offset'       => $offset,
+			'hide_empty'   => true,
+			/*
+			 * Limits aren't included in queries when hierarchical is set to true (by default).
+			 * See: https: //github.com/WordPress/WordPress/blob/5.3/wp-includes/class-wp-term-query.php#L558-L567
```

Comment:
If we want to use PHPDoc lingo here then use @link here (and a blank line before it haha finally!)